### PR TITLE
blog: Add June 2023 Updates post

### PR DESCRIPTION
**Pull request by ricekot**
<details>
<summary>Screenshot (click to expand)</summary>

image

</details>

### CODE REVIEW

File: site/content/blog/2023-07-05-zap-updates-june-2023/index.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+- akshath
+---
+
+Welcome to our monthly updates blog post!  
+There were a lot of incremental updates to ZAP in June 2023.
+
+## Highlights
+### Browser-Based Authentication Support in the AJAX Spider
+The AJAX Spider now supports crawling authenticated web pages with the browser-based authentication method.
+ZAP now runs regularly scheduled scans against OWASP Juice Shop that uses this mode of authentication. You can view more details about the scan [here](/docs/scans/juiceshop/).
```
Comment (psiinon) [inline site/content/blog/2023-07-05-zap-updates-june-2023/index.md:22]: I havnt changed it to use auth yet :)

File: site/content/blog/2023-07-05-zap-updates-june-2023/index.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+
+### Selenium 4
+All ZAP add-ons are now using [Selenium](https://www.selenium.dev/) v4.
```
Comment (psiinon) [inline site/content/blog/2023-07-05-zap-updates-june-2023/index.md:25]: Add something like: One significant improvement this brings is that messages written to stdout by browsers will no longer be shown in the ZAP logs.

File: site/content/blog/2023-07-05-zap-updates-june-2023/index.md [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+While anyone can use these images, we expect these to be useful for large organizations which may be hitting DockerHub's image pull limits.
+Read the [blog post](/blog/2023-06-13-ghcr-docker-images) for more details.
+
+### Community Scripts Tips (and Tricks)
+A new section was added to the community-scripts repository for people to share their ideas and usage tips for ZAP and its add-ons.
+Visit the [community-scripts repository](https://github.com/zaproxy/community-scripts/tree/main/other/tips) to learn more.
+
+## Ongoing Work
+### 2.13.0 Release
+We are actively working on getting ZAP 2.13.0 ready for release.
```
Comment (psiinon) [inline site/content/blog/2023-07-05-zap-updates-june-2023/index.md:38]: "You can track the progress on https://github.com/zaproxy/zaproxy/issues/7924" ?

**Comment by psiinon** [comment]
Thanks @ricekot - looks great. Just some minor suggestions.

File: site/content/blog/2023-07-05-zap-updates-june-2023/index.md (hunk 1 above)
Comment (psiinon) [reply site/content/blog/2023-07-05-zap-updates-june-2023/index.md:22]: But maybe mention that people should test any custom scripts and link to https://groups.google.com/g/zaproxy-users/c/qtXsiOWb7cg >

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by psiinon — APPROVED** [review]

**Comment by kingthorin** [comment]
Thanks for tackling this!

**Pull request merged**